babel/README.md

```diff
@@ -0,0 +1,8 @@
+# Babel application
+
+Full documentation about it [here](https://mochajs.org/#-require-module-r-module)
```

Comment:
Link has _nothing_ to do the application, but that Mocha will need `-r @babel/register` to test...

Reply:
I feel it's relevant and useful to point the user to the corresponding part of our documentation. 

If somebody is interested to know more about a particular detail/setup in a given example the docs offer that.